README.md

```diff
@@ -65,7 +65,7 @@ To show the Electron UI, with devTools open, and keep both alive, use the follow
 * `urlFilter` An optional predicate function, taking the current `url` as a parameter, and returning `true` or `false` as to whether or not to include it in the hunt. Second argument of `domainFiltered` is a bool stating whether or not the host matches (use it if you'd like to include that check in your filter)
 * `before` and `after` callbacks receive nightmare instance and if defined, must return it (see [examples/preAuth.js](https://github.com/justinjmoses/hounds/blob/master/examples/preAuth.js#L14-L26))
 * `screnshot` A function that given the current `url`, returns the path string of the PNG to save
-* `nightmare` All [nightmare 2.10.0 options](https://github.com/segmentio/nightmare/tree/2.10.0#nightmareoptions) are supported
+* `nightmare` All [nightmare 3.0.1 options](https://github.com/segmentio/nightmare/tree/3.0.1#nightmareoptions) are supported
```

Comment:
are you going to a piece on the new `consoleLevel` option?

Reply:
Yeah just been working on other stuff today.